sdk/core/core-amqp/src/cbs.ts

```diff
@@ -71,14 +72,26 @@ export class CbsClient {
   /**
    * Creates a singleton instance of the CBS session if it hasn't been initialized previously on
    * the given connection.
+   * @param options - Optional parameters that can be used to affect this method's behavior.
+   *    For example, `abortSignal` can be passed to allow cancelling an in-progress `init` invocation.
    * @returns Promise<void>.
    */
-  async init(): Promise<void> {
+  async init(options: { abortSignal?: AbortSignalLike } = {}): Promise<void> {
+    const { abortSignal } = options;
+    const initAbortMessage = "The init operation has been cancelled by the user.";
```

Comment:
In Service Bus, we chose to go with a generic message and not tie to the operation name. Could we consider that here?

Reply:
Sure. We are using custom abort error messages in other parts of core-amqp already. Mind if I standardize those as well then?

Reply:
Ah ok. In that case, we can do it in a separate PR covering both Event Hubs and core-amqp

Reply:
Sounds good. #13838 created to make sure I don't forget.